src/tests/multihost/ipa/conftest.py

```diff
@@ -30,6 +31,35 @@ def pytest_configure():
 # ====================  Function Scoped Fixtures ==============
 
 
+@pytest.fixture(scope='function')
+def bkp_and_config_for_subid_files(session_multihost, request):
```

Comment:
Why is this a function fixture and not a class fixture?

Reply:
becouse this fixture contains commands that need to execute in every test function

Reply:
Why?

Reply:
Changes done